p8Layouts/defaultPnorLayoutWithGoldenSide.xml

```diff
@@ -287,6 +289,7 @@ Layout Description
         <physicalOffset>0x1F60000</physicalOffset>
         <physicalRegionSize>0x90000</physicalRegionSize>
         <sha512Version/>
+        <readOnly/>
         <side>A</side>
         <ecc/>
```

Comment:
What about all the other partitions that need to be read only, to bring the list to parity with the list in p9?  All the code partitions would be read only, not just HBB/HBI right?